README.md

```diff
@@ -68,3 +68,28 @@ After these workarounds, you should finally be able to run ```bundle exec middle
 
 ### Other Troubleshooting tips
 If you're running into other problems, check [Troubleshooting.md](TROUBLESHOOTING.md) for other suggestions.
+
+# Maintainers
+
+This section will document some of the processes that members of the documentation team should adhere to.
+
+## Review period
+
+The two weeks preceeding the scheduled date of the next release is considered the review period of the Guides.
+
+It is only during this period that Pull Requests for the relevant milestone are to be merged in.
```

Comment:
"pull requests" is generally [not capitalized](https://help.github.com/articles/using-pull-requests/).